rts/lra/lra-proxy/api/src/main/resources/META-INF/beans.xml

```diff
@@ -2,5 +2,5 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee 
 		http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
-       bean-discovery-mode="annotated">
-</beans>
\ No newline at end of file
+       bean-discovery-mode="all">
```

Comment:
Is this a change from the default. or to the default? if you are moving to the default then no futher action, if changing from the default I would be interested by a comment as to what the nature of the change is